docs/develop/convert-xml-to-json-manifest.md

```diff
@@ -50,6 +50,11 @@ When you've added the files to the project, add **\<IconUrl\>** and **\<HighReso
 
 If your manifest has any **\<FunctionName\>** elements, make sure their values have fewer than 65 characters. The value of this element must exactly match the name of a function in a JavaScript or TypeScript file. If you change it in the manifest, be sure to change it in the code file too.
 
+### Ensure that your SSO add-in requests permissions 
+
+If your add-in uses Microsoft single sign-in, your add-in has a **\<Scopes\>** element that specifies the Azure Active Directory or Microsoft Graph permissions that the add-in needs. With the unified manifest, permissions must be requested in runtime code. Update your code as needed to request these permissions. The exact code depends on the architecture and authorization code libraries you are using. Typically, code requests permissions in a function that requests an access token. 
```

Comment:
```suggestion
If your add-in uses Microsoft single sign-on with the on-behalf-of (OBO) flow, your add-in has a **\<Scopes\>** element that specifies the Microsoft Graph or other API permissions that the add-in needs. With the unified manifest, permissions must be requested at runtime in the code. Update your code as needed to request these permissions. The exact code depends on the architecture and authorization code libraries you are using. Typically, code requests permissions in a function that requests an access token. 
```

Reply:
How do you request permission from the code?

Reply:
> How do you request permission from the code?

It depends on the framework being used. For example, in our [Office-Add-in-Microsoft-Graph-ASPNET](https://github.com/OfficeDev/Office-Add-in-samples/blob/main/Samples/auth/Office-Add-in-Microsoft-Graph-ASPNET/Office-Add-in-Microsoft-Graph-ASPNETWeb/Controllers/AzureADAuthController.cs) sample, it's done with the following C# code: 

```
 // Generate the parameterized URL for Azure login.
 string[] graphScopes = { "Files.Read.All", "User.Read" };
 var urlBuilder = clientApp.GetAuthorizationRequestUrl(graphScopes);
 urlBuilder.WithRedirectUri(loginRedirectUri.ToString());
 urlBuilder.WithAuthority(Settings.AzureADAuthority);
 urlBuilder.WithExtraQueryParameters("state=" + authState);
 var authUrl = await urlBuilder.ExecuteAsync(System.Threading.CancellationToken.None);
         
 // Redirect the browser to the login page, then come back to the Authorize method below.
 return Redirect(authUrl.ToString());
```